examples/rcbd_one_var.R

```diff
@@ -0,0 +1,41 @@
+wheat.data <- read.csv("Wheat_yield_data.csv", header = TRUE, stringsAsFactors = FALSE)
+
+library(car)  # for leveneTest()
+library(agricolae)  # for LSD.test()
+
+#Inform R that Block and Treatment are factors
+wheat.data$Block<-as.factor(wheat.data$Block)
+wheat.data$Treatment<-as.factor(wheat.data$Treatment)
+
+##The ANOVA##
+wheat.mod<-lm(Yield ~ Treatment + Block, wheat.data)
+anova(wheat.mod)
+
+##TESTING ASSUMPTIONS##
+#Generate residual and predicted values
+wheat.data$resids <- residuals(wheat.mod)
+wheat.data$preds <- predict(wheat.mod)
+wheat.data$sq_preds <- wheat.data$preds^2
+
+#Look at a plot of residual vs. predicted values
+plot(resids ~ preds, data = wheat.data,
```

Comment:
do you prefer this code to simply calling `plot(wheat.mod)`? To get this exact single plot you can call `plot(wheat.mod, 1)`.